### PR TITLE
qdevices: change requested-size type in json for -device

### PR DESCRIPTION
**Pull request by nickzhq**
ID: 2170865

### CODE REVIEW

**Comment by nickzhq** [comment]
Hello @mcasquer , please use this pr to test the failed case, thanks!
@zhencliu @YongxueHong Please take a look, thanks!

**Comment by nickzhq** [comment]
(1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.virtio_mem_numa_basic.q35: STARTED
 (1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.virtio_mem_numa_basic.q35: PASS (393.38 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Review by YongxueHong — APPROVED** [review]
LGTM.

File: virttest/qemu_devices/qdevices.py [hunk 1]
```diff
@@ -1197,6 +1197,10 @@ def _cmdline_json(self):
             elif val in ('off', 'no', "false") \
                     and key not in expect_string_val:
                 command_dict[key] = False
+            # requested-size from device("driver": "virtio-mem-pci")
+            elif key in ("requested-size"):
```
Comment (YongxueHong) [inline virttest/qemu_devices/qdevices.py:1201]:
```suggestion
            elif key in ("requested-size", ):
```
for example:
```
>>> key = "aa"
>>> key in ("aaaaa")
True
>>> key in ("aaaaa", )
False
```

**Review by zhencliu — APPROVED** [review]

File: virttest/qemu_devices/qdevices.py (hunk 1 above)
Comment (zhencliu) [reply virttest/qemu_devices/qdevices.py:1201]: This really matters, thanks @YongxueHong

Comment (nickzhq) [reply virttest/qemu_devices/qdevices.py]: Done

**Comment by nickzhq** [comment]
@YongxueHong Please check again, thanks!

**Pull request merged**